orangecontrib/timeseries/widgets/owspiralogram.py

```diff
@@ -399,7 +400,8 @@ def update_agg_combo(self):
                 new_aggs = [agg for agg in AGG_OPTIONS if AGG_OPTIONS[agg].time]
         self.combo_func.addItems(new_aggs)
 
-        self.agg_func = next(iter(new_aggs))
+        self.agg_func = self.agg_func if self.agg_func in new_aggs else next(
+            iter(new_aggs))
```

Comment:
Break if-else at else. Not having to use `\` isn't worth breaking the line unsemantically.

This would be better:

```python
        self.agg_func = self.agg_func if self.agg_func in new_aggs \
            else next(iter(new_aggs))
```

But in essence, the code does this:

```python
        if self.agg_func not in new_aggs:
            self.agg_func = next(iter(new_aggs))
```

`self.agg_func = self.agg_func if (...)` looks strange. :)

Reply:
For the third time, you're right. Very annoying. 😉